snapcraft/tests/test_lxd.py

```diff
@@ -367,16 +376,21 @@ def test_inject_snap_dangerous(self,
             {'name': 'core',
              'confinement': 'strict',
              'id': '2kkitQurgOkL3foImG4wDwn9CIANuHlt',
+             'channel': 'stable',
              'revision': '123'},
             {'name': 'snapcraft',
              'confinement': 'classic',
              'id': '',
+             'channel': 'edge',
              'revision': 'x1'},
         ]
 
         builder = self.make_cleanbuilder()
 
         builder.execute()
+        if hasattr(self, 'cross') and self.cross:
+            return
```

Comment:
I don't understand this return. Should it be a skip? A test without assertion is wrong.

Reply:
I was thinking the existing assertions didn't make sense... I added different ones now.

Reply:
The return still seems to be here, though.

Reply:
I figured it's redundant since it's the same in both cases... which in hindsight I should've said in the comment. I added it now anyway.